content/announcing-sigstore-java-1-0.md

```diff
@@ -0,0 +1,37 @@
++++
+title = "Announcing sigstore-java 1.0"
+author = "Appu Goundan (@loosebazooka)"
+date = "2024-09-00"
+draft = true
+type = "post"
+tags = ["sigstore","java","clients"]
++++
+
+# sigstore-java
+The sigstore-java project brings the Sigstore signing and verification paradigm to the Java ecosystem. It is built natively in Java and is easy to integrate with your Maven and Gradle builds or custom workflows.
+
+## 1.0 Stable Release
+Today, thanks to the support of the community and work of our contributors, we’re excited to announce a 1.0 stable release of the sigstore-java client. This includes a library([`dev.sigstore:sigstore-java`](https://central.sonatype.com/artifact/dev.sigstore/sigstore-java)) for programmatic access to the sigstore signing and verification APIs. It also includes signing plugins for maven([`dev.sigstore:sigstore-maven-plugin`](https://central.sonatype.com/artifact/dev.sigstore/sigstore-maven-plugin)) and gradle([`dev.sigstore.sign`](https://plugins.gradle.org/plugin/dev.sigstore.sign)) that integrate effortlessly with your build and publishing workflows.
+
+## Signatures on Maven Central
+Thanks to the team at Sonatype, Sigstore signature bundles (`*.sigstore.json`) are an accepted signature format on Maven Central. So you can start signing your Maven Central artifacts with Sigstore today. To ensure existing verification workflows and tooling continue to work for Java workflows PGP signing is currently still required for publishing to Maven Central.
+
+## Features
+The `sigstore-java` library is meant to provide a comprehensive API to sigstore infranstucture. Giving users access to both simple high level API while also making it possible to tweak the behavior according to your needs.
+
+The 1.0 ships with these features today
+- Programmatic signing and verifying using sigstore-java APIs
+- Easy signing using Maven and Gradle Sigstore plugins
+- Signing identity via browser workflow or from Github Actions id-token
```

Comment:
Does this support only GitHub Actions or other CI providers like GitLab?

Reply:
Only github actions for now. We'll add in the other providers in 1.1.0

Reply:
is there an issue to track that?

Reply:
I found https://github.com/sigstore/sigstore-java/issues/780 for Jenkins
I don't know if there are other ones for GitLab or others

Reply:
Jenkins is a little harder to support as it's mostly self hosted, I don't know how we'd get the oidc provider into fulcio. I'll add an ambient bug here: https://github.com/sigstore/sigstore-java/issues/795